### ISSUE TITLE
<Message is not decryptable on this device>

### ISSUE THREAD

**Issue by namelessperson0** [ItsAditya-xyz/DeSo.py#20]
When a private message is sent though this library, the sender is shown the below message in the inbox from bitclout.com and diamondapp.com
`<Message is not decryptable on this device>
`
The receiver receives the correct message though. But the sender has issues in seeing the message sent.

**Comment by ItsAditya-xyz** [comment]
hmm, that's weird. I am having that problem as well
![image](https://user-images.githubusercontent.com/55331140/173386276-72b0a2c9-63d3-4dee-aa2b-27629e44ced0.png)

likely because the DM is not being encrypted before sending...

**Comment by namelessperson0** [comment]
Note that receiver is able to read the message without issue. It is only the sender having the issue.

**Comment by ItsAditya-xyz** [comment]
i think it's an issue with the frontends itself (if the receiver can see the msg, than it's obvious the sender can see it too.
btw, the library doesn't really encrypts the message before sending so i think that might be the issue

**Comment by namelessperson0** [comment]
But messages sent in frontend do not have this issue.

**Comment by ItsAditya-xyz** [comment]
yup, because those messages are not encrypted. You can see the payload request while sending a msg

**Comment by namelessperson0** [comment]
bitclout.com uses "EncryptedMessageText" in post data while this python library uses "MessageText".

In deso-protocol nodejs package, `this.identity.encrypt` is used to encrypt the message at https://github.com/deso-protocol/deso-workspace/blob/58095ec99eb2010feba55c39e6507bd14d7529b1/libs/deso-protocol/src/lib/social/Social.ts#L44

That encrypt function is defined in https://github.com/deso-protocol/identity/blob/dc744ab49944409b28f36ffe039d39cacf16e92b/src/app/signing.service.ts#L28

I think we will need this package https://github.com/ecies/py for encrypting the message.

**Comment by ItsAditya-xyz** [comment]
I see. I will try figure that out. Feels free to create a PR to solve this!
